### PR TITLE
jgrss/dask version

### PR DESCRIPTION
**Pull request by jgrss**
## What is this PR changing?
Pins the maximum version of `Dask` and `distributed`. Versions > `v2022.8.0` cause a cryptic error when using `xarray.save_mfdataset()`.

Fixes a bug with Landsat angle creation, where if sampling >1 was used then the angle raster extent was incorrect (i.e., too small).

### CODE REVIEW

**Comment by jgrss** [comment]
@mmann1123 do you want 👀 on this?

File: src/geowombat/radiometry/angles.py [hunk 1]
```diff
@@ -482,9 +482,8 @@ def open_angle_file(
     band_pos: int,
     nodata: T.Union[float, int],
     subsample: int,
-    ref_res: T.Sequence[float],
 ) -> xr.DataArray:
-    new_res = subsample * ref_res[0]
+    new_res = subsample * 30.0
```
Comment (jgrss) [inline src/geowombat/radiometry/angles.py:486]: Landsat angle creation fix

File: src/geowombat/radiometry/angles.py [hunk 2]
```diff
@@ -576,6 +567,47 @@ def postprocess_espa_angles(
     )
 
 
+def landsat_angle_prep(
```
Comment (jgrss) [inline src/geowombat/radiometry/angles.py:570]: All other changes in Landsat angle creation are just reformatting.

File: tests/test_rad.py [hunk 3]
```diff
@@ -20,6 +63,82 @@ def test_landsat_metadata(self):
...
+                # Used to generate test data
+                # save_angle_data(angle_array_resamp_da, angle_paths_in.solar, nodata)
+                data = (
+                    angle_array_resamp_da.fillna(nodata)
+                    .gw.assign_nodata_attrs(nodata)
+                    .astype('int32')
+                )
+                # Check against a full solar angle image
+                with gw.open(l7_225078_20110306_SZA, chunks=data.gw.row_chunks) as src:
+                    self.assertTrue(src.equals(data))
```
Comment (jgrss) [inline tests/test_rad.py:111]: This should catch future issues, as it compares against an angle raster that correctly covers a full WRS2 path/row.

**Review by mmann1123 — APPROVED** [review]
Looks good sorry for delay

**Pull request merged**